controllers/apps/configuration/configuration_controller.go

```diff
@@ -113,6 +114,13 @@ func (r *ConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		reqCtx.Log.Info("cluster is deleting, skip reconcile")
 		return intctrlutil.Reconciled()
 	}
+	// if cluster is paused, pausing reconfigure as well
+	if fetcherTask.ClusterObj != nil {
```

Comment:
provider a utility function for configuration and reconfigure reconciler

Reply:
And it should check the object the reconciler managed for, but not the cluster object.